### DISCUSSION TITLE
Outdated policy in Fleet

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#12448] [General]
Good Afternoon,

I have updated a policy in Fleet and waited several hours.  All of the hosts stay in the "Outdated policy" mode, but show that they are communicating with the manager.  Firewalls don't appear to be blocking as these hosts have bee communicating for a while.

I also tried removing the policy and reapplying, now the agent is in "Updating" rather than "Healthy"

Am I missing something?

Currently running 2.4.50

**Comment by oneCrazyAdmin** [comment]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/78620004/851f7c80-f8af-402a-a494-6353702606d5)

I do have this in my Fleet Console as well.
'

**Reply by cm-ops** [reply]
What does `elastic-agent status` from the manager's CLI return?

**Reply by oneCrazyAdmin** [reply]
This is what it shows...

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/78620004/549301ba-6534-4a1a-923d-017ad3958491)

Just and update... I found the so-elastic-fleet-reset and so-elastic-fleet-setup commands.  I ran those a few times and I now have the Fleet Server again.  However, it wiped out all of the out of the box policies with the integrations.

**Reply by cm-ops** [reply]
Check `/opt/so/state/eaintegrations.txt` if that file is present, remove it and run `sudo so-elastic-fleet-integration-policy-load`.

**Reply by oneCrazyAdmin** [reply]
Thanks!  I did that and they are all showing back up again!
